src/components/dls/QuranWord/MadaniWordText.tsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import styled from 'styled-components';
+import { decamelizeKeys } from 'humps';
 
 type MadaniWordTextProps = {
   text: string;
+  fontVersion: string;
```

Comment:
nit: should we just call this prop `font`?

Reply:
Sure